R/io-parsers.R

```diff
@@ -413,17 +422,24 @@ parse_mixcr <- function(.filename, .mode) {
   # cloneCount - clonalSequence - nSeqCDR3
   # cloneCount - targetSequences - nSeqImputedCDR3
   # cloneCount - targetSequences - nSeqCDR3
-  if ("targetsequences" %in% table.colnames) {
-    if ("nseqimputedcdr3" %in% table.colnames) {
-      .nuc.seq <- "nseqimputedcdr3"
+
+  for (cdr in c("cdr1", "cdr2", "cdr3")) {
+    # assign variables .nuc.seq.cdr1, .nuc.seq.cdr2, .nuc.seq.cdr3
+    cdr_var <- paste0(".nuc.seq.", cdr)
+    if ("targetsequences" %in% table.colnames) {
+      if (paste0("nseqimputed", cdr) %in% table.colnames) {
+        assign(cdr_var, paste0("nseqimputed", cdr))
```

Comment:
Same as in upper comment `cdr_var<-paste0("nseqimputed", cdr)`

Reply:
assign and <- are not the same; assign means that cdr_var contains name of variable that should be assigned